src/v/coproc/api.cc

```diff
@@ -59,12 +64,14 @@ ss::future<> api::start() {
     co_await _pacemaker.invoke_on_all(&coproc::pacemaker::start);
     _listener = std::make_unique<wasm::event_listener>();
```

Comment:
I'm confused about why all these things (partition manager, reconcile backend, pacemaker, etc...) seem to be started if coproc is not enabled